docs/getting_started/extending_flyte.md

```diff
@@ -26,7 +26,7 @@ the Flyte type system and Flyte tasks.
 
 ## Customizing Flyte Types
 
-Flyte has a {ref}`rich type system <flyte_type_system>` that automatically
+Flyte has a {ref}`rich type system <data_types_and_io>` that automatically
```

Comment:
revert ref to `flyte_type_system` to avoid breaking changes with other docs sites